cdn/dev/keyboard-search/dedicated-landing-pages.js

```diff
@@ -0,0 +1,66 @@
+//
+// Meta search results for dedicated landing pages
+//
+
+const dedicatedLandingPages = [
+  {
+    id: "/amharic",
+    terms: ['amharic', 'ge\'ez', 'geez', 'ethiopic'],
+    description: "Amharic keyboards recommended by the Keyman team",
```

Comment:
Do you want to include `አማርኛ ይጻፉ` in the description?

Reply:
Not at this point. It'd be the only one with non-English text in the description!